python/dgl/distributed/graph_partition_book.py

```diff
@@ -247,6 +277,16 @@ def partid(self):
             The partition id of current machine
         """
 
+    @property
+    def ntypes(self):
+        """Get the list of node types
+        """
+
+    @property
+    def etypes(self):
+        """Get the list of edge types
+        """
```

Comment:
raise NotImplementedError?

Reply:
what is the practice in other places?

Reply:
I think it'll be raising NotImplementedError if you don't use abstract classes.